.github/workflows/pylint.yml

```
@@ -21,11 +23,19 @@ jobs:
         python get-pip.py
         pip install pylint==1.1.0
         pip install astroid==1.2
-        
+
     - name: Run Pylint Checks
       run: |
-        pylint --rcfile=.pylintrc convert2rhel/
-        
+         function interpreted_pylint() {
+            local es=0
+            pylint --rcfile=.pylintrc convert2rhel/; es=$?
```

```suggestion
            pylint --rcfile=.pylintrc convert2rhel/ || es=$?
```
This code is executed with bash pipefail which means that if we use `;` here, the whole application will fail on non-zero return code. To ensure that it doesn't fail through we need to use `or` and execute our assignment on the right hand side.